.changeset/famous-seas-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': minor
+---
+
+added config for `_routes.json` generation
```

Comment:
```suggestion
Adds a new config option for `_routes.json` generation: `routes.strategy`.
```
I personally like listing the new option here! Will make it easy for people to know what to look for if they're reading the changelog!